### DISCUSSION TITLE
Conversation style "precise" is not working correctly

### DISCUSSION THREAD

**Discussion by tranngocminhhieu** [acheong08/EdgeGPT#442] [Q&A]
Hello developers

With the same question and precise mode, when chatting directly on the browser, Bing answered exactly what I wanted. It only answers 1 of 3 options: "Positive", "Negative", "Normal".

But when I use it on Python (selected conversion style is precise) the result is not as expected. It responds very verbosely like the other 2 modes.

Please check for me what is wrong in conversion style precise in your code.

Thank you!

Here is the code that I used:

```
import json
import asyncio
import nest_asyncio
nest_asyncio.apply()
from EdgeGPT import Chatbot, ConversationStyle

cookies = json.loads(open('bing_cookie.json', encoding='utf-8').read())

async def main(prompt):
    bot = await Chatbot.create(cookies=cookies) # Passing cookies is optional
    print(await bot.ask(prompt=prompt, conversation_style=ConversationStyle.precise))
    await bot.close()

prompt = '''If this is an article about your company, how might it affect investor sentiment (stock buyers)? Answer only "Positive", "Negative", "Normal", not verbose.
Article title: Novaland is in debt, doesn't pay employees'''

result = asyncio.run(main(prompt=prompt))
```

**Comment by Nu11Const** [comment]
Yeah,I'm trying to change it.
